test/unit/custom-style-late.html

```diff
@@ -14,19 +14,48 @@
   <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../../web-component-tester/browser.js"></script>
   <link rel="import" href="../../polymer.html">
-  <link rel="import" href="custom-style-late-import.html">
-
 </head>
 <body>
-  <input id="input" is="x-input">
+
+  <custom-style>
+    <style is="custom-style">
+      x-input {
+        border: 4px solid red;
+        @apply --cs-blue;
+      }
+    </style>
+  </custom-style>
+
+  <template id="late">
+  <custom-style>
+    <style is="custom-style">
+      :root {
+        --cs-blue: {
+          border: 8px solid blue;
+        }
+        ;
```

Comment:
I think this was trying to test some broken syntax, but I'm fine cleaning it up.